screenpy/resolutions/matches.py

```diff
@@ -17,14 +24,21 @@ class Matches:
         )
```

Comment:
Can you add more examples with the new options, like `re.compile` and a plain string?